buildSrc/src/main/groovy/io.micronaut.build.internal.nats-module.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'io.micronaut.build.internal.module'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compileOnly 'com.google.code.findbugs:jsr305' // for "warning: unknown enum constant When.MAYBE"
+
+    testRuntimeOnly 'ch.qos.logback:logback-classic:1.2.7'
+}
+
+test {
+    useJUnitPlatform()
```

Comment:
:x: This is redundant

Reply:
I've removed the whole testing block as it's redundant.